06.bowling_object/game.rb

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+class Game
+  def initialize(scores)
+    @frames = [Frame.new(0)]
```

Comment:
最初のフレームだけ特別扱いされてるので、ここは

```suggestion
    @frames = []
```

として、他のフレームと同じ枠組みの中でnewさせたいですね

```ruby
    frame = @frames.last
    if frame.nil? || frame.finished?
      @frames << Frame.new(@frames.size)
    end
```